drivers/interrupt_controller/exti_stm32.c

```diff
@@ -214,45 +151,16 @@ void stm32_exti_enable(int line)
 		case 22:
 			irqnum = RTC_WKUP_IRQn;
 			break;
+#endif
+#if defined(CONFIG_SOC_SERIES_STM32F7X)
 		case 23:
 			irqnum = LPTIM1_IRQn;
 			break;
+#endif
+		default:
+			return;
 		}
 	}
```

Comment:
case F2/F4/F7 seems in line with previous code.
But I'm not sure about F1/F3. In these case if line is above 15 what happens?

Reply:
@avisconti, I reworked a bit the change to be closer to previous code for L4.
For F1/F3, line > 15, code was obviously wrong, also we missed handling for not supported lines for F2/F4/F7 so reporting an error now.

Reply:
> For F1/F3, line > 15, code was obviously wrong, also we missed handling for not supported lines for F2/F4/F7 so reporting an error now.

OK, got it now.